### PR TITLE
To disable WebVideo

### PR DESCRIPTION
**Pull request by PeterWangIntel**
With flag "-Ddisable_web_video=1" the size is reduced by 180K.

### CODE REVIEW

**Comment by PeterWangIntel** [comment]
@d0ngjun

File: Source/core/html/HTMLMediaElement.cpp [hunk 1]
```diff
@@ -3437,7 +3437,9 @@ void HTMLMediaElement::createMediaPlayer()
 
     closeMediaSource();
 
+#ifdef DISABLE_WEB_VIDEO
```
Comment (d0ngjun) [inline Source/core/html/HTMLMediaElement.cpp]: ifndef

Comment (PeterWangIntel) [inline Source/core/html/HTMLMediaElement.cpp]: sorry

**Comment by PeterWangIntel** [comment]
sorry wrong patch

**Pull request closed without merging**